### ISSUE TITLE
Hero method argument is not converting correctly to OW Workshop

### ISSUE THREAD

**Issue by LionariusPy** [ItsDeltin/Overwatch-Script-To-Workshop#121]
Code:
```
rule: "Create AI"
Event.OngoingPlayer
if (IsCommunicating(EventPlayer(), Communication.Acknowledge))
{
    CreateAI(Hero.Soldier76, TeamOf(), PositionOf());
}

public void CreateAI(Hero hero, Team team, define position)
{
    CreateDummyBot(hero, team, -1, position, Vector.Zero);
}
``` 

``` CreateDummyBot(hero, team, -1, position, Vector.Zero); ```
gets converted to 
```
Create Dummy Bot(Soldier: 76, Team Of(Event Player), -1, Player Variable(Event Player, position), Vector(0, 0, 0));
```
instead of
```
Create Dummy Bot(Hero(Soldier: 76), Team Of(Event Player), -1, Player Variable(Event Player, position), Vector(0, 0, 0)); 
```

**Comment by derwangler** [comment]
As a workaround, use `ref define hero` instead of `Hero hero` in the parameter:
```cs
public void CreateAI(ref define hero, Team team, define position)
{
    CreateDummyBot(hero, team, -1, position, Vector.Zero);
}
```
Outputs correctly:
`Create Dummy Bot(Hero(Soldier: 76), Team Of(Event Player), -1, Player Variable(Event Player, position), Vector(0, 0, 0));`

**Comment by ItsDeltin** [comment]
The `Hero`, `Team`, `Gamemode`, and `Map` enums should be changed to return the actual value all the time rather than the enum used to obtain the values.

**Comment by ItsDeltin** [comment]
Fixed in [v1.3](https://github.com/ItsDeltin/Overwatch-Script-To-Workshop/releases/tag/v1.3).
